modules/sdk-core/src/bitgo/wallet/iWallets.ts

```diff
@@ -14,15 +14,24 @@ export interface GetWalletOptions {
   id?: string;
 }
 
-export interface GenerateMpcWalletOptions {
-  multisigType: 'onchain' | 'tss' | 'blsdkg';
+export interface GenerateBaseMpcWalletOptions {
+  multisigType: 'tss' | 'blsdkg';
   label: string;
-  passphrase?: string;
-  originalPasscodeEncryptionCode?: string;
-  enterprise?: string;
+  enterprise: string;
   walletVersion?: number;
+}
+
+export interface GenerateMpcWalletOptions extends GenerateBaseMpcWalletOptions {
+  passphrase: string;
+  originalPasscodeEncryptionCode?: string;
```

Comment:
Would be nice to add a js doc describing what this param means, as for other readers its not always clear/obvious.

Reply:
updated js docs on top of the generateWallet method